test/util.jl

```diff
@@ -1,4 +1,5 @@
 using DSP, Base.Test, Compat
+using SpecialFunctions
```

Comment:
`besseli` has used in the function `kaiser` at src/windows.jl,
so it could be `import SpecialFunctions.besseli` to the module `Windows`.